pkg/oc/cli/cmd/rollout/latest.go

```diff
@@ -199,7 +203,7 @@ func (o RolloutLatestOptions) RunRolloutLatest() error {
 		fmt.Fprintf(o.out, fmt.Sprintf("%d", dc.Status.LatestVersion))
 		return nil
 	} else if len(o.output) > 0 {
-		return o.printer.PrintObj(dc, o.out)
+		return o.printObj(dc, info.Mapping, o.out)
```

Comment:
@soltysh normally, requesting generic output (like json or yaml) from a command would act as a `--dry-run`, without actually performing any changes to objects on the server-side.

I was not sure how to approach this with the `rollout latest` command, given that it already supported the `"revision"` output format, which is printed after the dc has been rolled out.

The new output behavior of this PR remains consistent with what was already there. If `json` or `yaml` or any other output format is requested, then the rollout still happens, and then the dc is printed out in the format specified.

Thoughts welcome on this.

Reply:
Honestly, I'd prefer to get rid of the output in commands like this one. It was a mistake we ever decided to add those.